src/07_slices.py

```diff
@@ -12,26 +12,33 @@
 a = [2, 4, 1, 7, 9, 6]
 
 # Output the second element: 4:
-print()
+i = slice(1, 2)
+print(a[i])
 
 # Output the second-to-last element: 9
-print()
+i = slice(4, 5)
+print(a[i])
 
 # Output the last three elements in the array: [7, 9, 6]
-print()
+i = slice(3, 6)
+print(a[i])
 
 # Output the two middle elements in the array: [1, 7]
-print()
+i = slice(2, 4)
+print(a[i])
 
 # Output every element except the first one: [4, 1, 7, 9, 6]
-print()
+i = slice(2, 6)
+print(a[i])
 
 # Output every element except the last one: [2, 4, 1, 7, 9]
-print()
+i = slice(0, (len(a) - 1))
+print(a[i])
 
 # For string s...
 
 s = "Hello, world!"
 
 # Output just the 8th-12th characters: "world"
-print()
\ No newline at end of file
+i = slice(7, 12)
+print(s[i])
```

Comment:
checkout https://stackoverflow.com/questions/509211/understanding-slice-notation/509295#509295 and see if you can rewrite the slices using that syntax